apps/accounts/templates/provider_registration/locations.html

```diff
@@ -0,0 +1,114 @@
+{% extends "base.html" %}
+{% load i18n static %}
+
+{% block head %}
+
+{% endblock %}
+
+{% block content %}
+<div class="container mx-auto">
+    {{ wizard.form.media }}
+    <!-- Initialize a ConvenientFormset -->
+    <script>
+    window.addEventListener('load', function (event) {
+        new ConvenientFormset({
+            'formsetPrefix': '{{ wizard.form.prefix }}',
+            'formsContainerSelector': '#convenient-formset',
+            'formSelector': '.convenient-form',
+
+            'canAddForms': true,
+            'addFormButtonSelector': '#add-form-button',
+            'emptyFormSelector': '#empty-form .convenient-form',
+
+            'canDeleteForms': true,
+            'deleteFormButtonSelector': '#delete-form-button',
+        });
+    });
+</script>
+
+<p>Step {{ wizard.steps.step1 }} of {{ wizard.steps.count }}</p>
+
+<section class="prose my-10">
+	<h1>Which parts of the world is your organisation active in?</h1>
+
+    <p>Starting with your main headquarters in your home country, list the locations your organisation is actively selling services in. We collect this information to help people looking for services available in a given country or region.</p>
+
+    <p>If your organisation runs services in a datacentre in given city, list the city and country for each region you offer. If you operate from a single datacentre or region, but have offices selling services in different countries, please list the cities you operate in.</p>
+
+</section>
+
+{% if form.is_multipart %}
+    <form enctype="multipart/form-data" method="post" action="" class="form__table">
+{% else %}
+    <form method="post" action="" class="form__table">
+{% endif %}
+
+    {% csrf_token %}
+    {{ wizard.management_form }}
+    {{ wizard.form.management_form }}
+
+    <div id="convenient-formset">
+
+    {% for form in wizard.form.forms %}
+    <div class="convenient-form bg-neutral-200 rounded-xl my-6 p-4">
+        <div class="form__container--location">
+            {{ form.non_field_errors }}
+
+            {% comment %}
+            Iteration over all fields in the location form. This is
+            equivalent to listing a span for each form.field_name
+            where field_name is in: ["title", "link", "file", "type",
+            "public", "description"]
+            {% endcomment %}
```

Comment:
Small nitpick: these fields `[["title", "link", "file", "type", "public", "description"]`are for evidence and not for location. This comment can be either removed or updated